test/test.js

```diff
@@ -125,13 +129,13 @@ test('two emails with different handlers', function(t) {
 			sendmail(recp, subject+"0", body+"0", cb);
 		},
 		function (res,cb) {
-			mailServer.bind(recp,handler0(cb));
+			mailServer.bind(recp,handler0Factory(cb));
```

Comment:
Calling `handler*factory` now assigns the created handler directly to `handler*`, which passes the equality check in `unbind`. Before, different function instances caused the equality check (`f !== g`) to fail.